### PR TITLE
Allow user interaction in SwipeItemView

### LINKED ISSUE #8779: [Bug][iOS][Android] Entry in custom SwipeItemView can't gain focus
### Description

I've defined a custom SwipeItemView that contains an Entry. When the SwipeItemView is revealed, it's impossible to focus in the Entry. Attempting to do so closes the SwipeItemView.

I realise this may be unsupported behavior, but I need to know if it is.

Note: It's possible to give the Entry focus with the TAB key on iOS.

### Steps to Reproduce

1. Run the attached sample.
2. Browse to the SwipeView custom SwipeItem.
3. Swipe from the right on "What's 2+2?".
4. Give focus to the Entry with the mouse/touch.

### Expected Behavior

The Entry gains focus and can receive input.

### Actual Behavior

The SwipeViewItem closes.

### Basic Information

- Version with issue: 4.4-pre3
- IDE: VSMac 8.3.10
- Platform Target Frameworks: 
  - iOS:  13.2
  - Android: API 28

### Reproduction Link

[SwipeViewDemos.zip](https://github.com/xamarin/Xamarin.Forms/files/3932438/SwipeViewDemos.zip)

### PR DESCRIPTION
**Pull request by jsuarezruiz**
### Description of Change ###

Allow user interaction in SwipeItemView.

### Issues Resolved ### 

- fixes #8779

### API Changes ###
 
 None

### Platforms Affected ### 

- iOS
- Android

### Behavioral/Visual Changes ###

None

#### Before
 Entry in custom SwipeItemView can't gain focus.

#### After

Android
issue8779-droid-after
iOS
issue8779-ios-after

### PR Checklist ###

- [x] Targets the correct branch
- [x] Tests are passing (or failures are unrelated)

### CODE REVIEW

File: Xamarin.Forms.Platform.Android/Renderers/SwipeViewRenderer.cs [hunk 1]
```diff
@@ -228,9 +233,29 @@ public override bool OnTouchEvent(MotionEvent e)
...
+				case MotionEventActions.Down:
+					_downX = e.RawX;
+					_downY = e.RawY;
+					_initialPoint = new APointF(e.GetX() / _density, e.GetY() / _density);
+					break;
+				case MotionEventActions.Move:
+					float x = Math.Abs((_downX - e.GetX()) / _density);
+					float y = Math.Abs((_downY - e.GetY()) / _density);
+
+					if (x > 10f || y > 10f)
```
Comment (VladislavAntonyuk) [inline Xamarin.Forms.Platform.Android/Renderers/SwipeViewRenderer.cs]: Please move 10f to constants

Is it worth to create a dependency property so the user can set the minimum delta to process swiping interactions?

Comment (jsuarezruiz) [reply Xamarin.Forms.Platform.Android/Renderers/SwipeViewRenderer.cs]: The value is used in several places, I have created a constant.
About creating a dependency property, there are several ideas for properties that we could add in future versions (`SwipeThreshold`, `SwipeMinimumDelta`, etc.). Remember that you can create a Spec with your proposal!.

File: Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue8779.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+namespace Xamarin.Forms.Controls.Issues
+{
+#if UITEST
+	[Category(UITestCategories.SwipeView)]
+#endif
+#if APP
+	[XamlCompilation(XamlCompilationOptions.Compile)]
+#endif
+    [Preserve(AllMembers = true)]
+    [Issue(IssueTracker.Github, 8779, "[iOS][Android] Entry in custom SwipeItemView can't gain focus", PlatformAffected.Android)]
```
Comment (jfversluis) [inline Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue8779.xaml.cs]:
```suggestion
    [Issue(IssueTracker.Github, 8779, "[iOS][Android] Entry in custom SwipeItemView can't gain focus", PlatformAffected.Android | PlatformAffected.iOS)]
```

**Comment by samhouts** [comment]
@jsuarezruiz Please rebase to fix conflicts! Thanks!

Failing tests are not related.

**Pull request merged**